Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WPF/Microsoft.Toolkit.Win32.UI.Controls.WPF.csproj

```diff
@@ -0,0 +1,102 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net462</TargetFramework>
+    <RootNamespace>Microsoft.Toolkit.Win32.UI.Controls.WPF</RootNamespace>
+    <AssemblyName>Microsoft.Toolkit.Win32.UI.Controls.WPF</AssemblyName>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+
+    <Title>Windows Community Toolkit Win32 Controls</Title>
+    <Description>This library provides XAML user controls for WPF. It is part of the Windows Community Toolkit.</Description>
+    <PackageTags>WebView Win32 WPF</PackageTags>
+
+    <!-- Need Metadata contract 6.0 or later -->
+    <!-- First introduced in 17110: Insider Preview (Fast) -->
+    <!-- Most people will have RTM 17134: April 2018 Update -->
+    <TargetPlatformMinVersion>10.0.17134.0</TargetPlatformMinVersion>
+    <!--
+    When changing this value, ensure the SDK is installed with the build process.
+    Need to check several files:
+      - /.vsts-ci.yml
+      - /.vsts-pr.yml
+
+    This also needs to be installed on your local machine. Can do this with PowerShell:
+      ./build/Install-WindowsSDKISO.ps1 17709
+    -->
+    <TargetPlatformVersion>10.0.17709.0</TargetPlatformVersion>
```

Comment:
The build seems to be failing, might be a mismatch between the insider sdk we install as part of the script. Could you update the project to target 17713 (here and in the other csproj - we should probably factor this out in directory.props to make it easier)

Reply:
I haven't had to mess with a Directory.Build.props file before...  
Looks like it's a matter of updating 
 
    <DefaultTargetPlatformVersion>17134</DefaultTargetPlatformVersion>
    <DefaultTargetPlatformMinVersion>15063</DefaultTargetPlatformMinVersion>

to 

    <DefaultTargetPlatformVersion>17713</DefaultTargetPlatformVersion>
    <DefaultTargetPlatformMinVersion>17134</DefaultTargetPlatformMinVersion>

and... removing the lines you highlight above from that csproj and others like them?

Anything else?

Reply:
Commit's up.